docs/sdks/client-sdks/react-native/initialization.mdx

```diff
@@ -0,0 +1,305 @@
+---
+title: Initialization
+sidebar_position: 3
+---
+
+import ApiOptionRef from '@site/src/components/ApiOptionRef';
+
+The Eppo JavaScript SDK is easy to initialize while offering robust customization options, making it adaptable to various use cases such as offline mode, custom caching requirements, and ultra-low-latency initialization.
+
+## Initialize the SDK
+
+To complete basic initialization, you only need to provide an SDK key. [Create an SDK key](/sdks/sdk-keys) if you don't already have one. 
+
+```javascript
+import { init } from "@eppo/react-native-sdk";
+
+await init({
+  apiKey: 'YOUR_SDK_KEY',
+});
+``` 
+
+## Use the SDK instance
+After initialization, you can use the SDK instance by importing the SDK at the scope you will use it in and calling `getInstance()`. You can then use the SDK instance to assign a variation to a subject using the `get*Assignment` functions.
+
+```javascript
+import * as EppoSdk from "@eppo/react-native-sdk";
+
+const eppoClient = EppoSdk.getInstance();
+const user = getCurrentUser();
+
+```
+
+
+## Advanced Configuration
+Basic initialization is great for most use cases, but the SDK provides options that you can use during initialization to customize the behavior of the SDK.
```

Comment:
We also have a `baseUrl` option which is where the SDK fetches configurations from. Some customers override this went they proxy configuration requests.